mycroft/client/speech/hotword_factory.py

```diff
@@ -77,18 +89,36 @@ def __init__(self, key_phrase="hey mycroft", config=None, lang="en-us"):
         self.lang = str(self.config.get("lang", lang)).lower()
 
     def found_wake_word(self, frame_data):
+        """Check if wake word has been found.
+
+        Checks if the hotword has been found. Should reset any internal
+        tracking of the hotword state.
+
+        Arguments:
+            frame_data (binary data): Deprecated. Audio data for large chunk
+                                      of audio to be processed. This should not
+                                      be used to detect audio data instead
+                                      use update() to incrementaly update audio
+        Returns:
+            (bool) True if a hotword was detected, else False
+        """
```

Comment:
Similar to the above, I think we should use one term "wake word" vs "hotword" unless we have a specific reason to use the other. We already use the two interchangeably in the code base which I think we should address at some point, but that's beyond the scope of this PR.

I think we should use "wake word" when referring to listening activation, and "hotword" when we go down the route of arbitrary hotword detection like "stop".